cores/cv32e40p/user_manual/source/exceptions_interrupts.rst

```diff
@@ -89,6 +89,7 @@ CV32E40P can trigger an exception due to the following exception causes:
 +----------------+---------------------------------------------------------------+
 
 The illegal instruction exception and M-Mode ECALL instruction exceptions cannot be disabled and are always active.
+The core raises illegal exceptions for any explicit RISC-V ISA defined case, and for any instruction encodings that are not defined in the CV32E40P core with respect to its specified parameters.
```

Comment:
The core raises an illegal instruction exception for any instruction in the RISC-V privileged and unprivileged specifications that is explicitly defined as being illegal, as well as for any instruction that is left undefined in these specifications unless the instruction encoding is configured as a custom CV32E40P instruction for specific parameter settings as defined in  (see :ref:`custom-isa-extensions`).